e2eTests/e2eTestPrompts.md

```diff
@@ -125,6 +125,8 @@ This file contains prompts used for end-to-end testing to ensure each tool is in
 | azmcp-keyvault-certificate-create | Create a new certificate called <certificate_name> in the key vault <key_vault_account_name> |
 | azmcp-keyvault-certificate-get | Show me the certificate <certificate_name> in the key vault <key_vault_account_name> |
 | azmcp-keyvault-certificate-get | Show me the details of the certificate <certificate_name> in the key vault <key_vault_account_name> |
+| azmcp-keyvault-certificate-import | Import the certificate in file <file_path> into the key vault <key_vault_account_name> |
+| azmcp-keyvault-certificate-import | Import a certificate into the key vault <key_vault_account_name> using the name <certificate_name> |
```

Comment:
I think we may want to transition to make these better, like the ones you have in prompts.

Reply:
You mean using a JSON file?

Reply:
I just mean, we now have two places for prompts.  Can we combine?